src/lib.rs

```diff
@@ -148,7 +148,7 @@
 //!     }
 //!
 //!     // Define additional `ErrorKind` variants. The syntax here is
-//!     // the same as `quick_error!`, but the `from()` and `cause()`
+//!     // the same as `impl_error_chain_kind!`, but the `from()` and `cause()`
```

Comment:
Oh, I think I realized that the implementation in the macro wasn't identical to the comment.  In fact, what is implemented inside the errors matcher of the macro is a bit above my understanding of the crate.  I've put in general documentation as my limited understanding permits, but am open to better verbiage.